app/views/OverviewPageView.scala.html

```diff
@@ -126,7 +147,18 @@ <h2 class="app-task-list__section">
                     </span>
                 </li>
             }
+
         </ol>
+
+            @if(!isInYear && isAnIncomeSourceDefined(incomeSources)){
```

Comment:
Are we sure we need to be hiding this paragraph if there are no updates? The prototype does not seem to reflect this
<img width="658" alt="Screenshot 2021-09-22 at 11 33 35" src="https://user-images.githubusercontent.com/28566263/134328710-e7644e11-70e0-4f0e-b781-5f791e77703f.png">
<img width="664" alt="Screenshot 2021-09-22 at 11 27 04" src="https://user-images.githubusercontent.com/28566263/134328738-eb0fab26-1c06-4b8e-9f16-3a1508def5ae.png">

.

Reply:
I set it to not show as that is what was previously happening with that section. I did want to ask Sara about it but she is currently on annual leave. I'll remove it however as you're right it's not like that on the prototype
<img width="822" alt="Screenshot 2021-09-22 at 11 44 33" src="https://user-images.githubusercontent.com/60218795/134343320-98aff72a-b783-49e3-851d-f61da3128ea2.png">
<img width="926" alt="Screenshot 2021-09-22 at 11 44 08" src="https://user-images.githubusercontent.com/60218795/134343348-efde0ff4-61cf-4a23-af52-909e6b5ec34d.png">